app/smc/prj.conf

```diff
@@ -23,6 +23,7 @@ CONFIG_SMBUS=y
 
 # protobufs
 CONFIG_NANOPB=y
+CONFIG_TT_FWTABLE_MOCK=y
```

Comment:
Note: this shouldn't be hard-coded in `app/smc/prj.conf`, but tacked-on to certain tests that might have state lingering from previous flash operations.

Reply:
Fixed by adding to specific `testcase.yml`, `samples.yml` files, and defaulting to y with `CONFIG_ZTEST=y`.